plugins/anchor-links/index.test.js

```diff
@@ -65,10 +184,31 @@ describe('anchor-links', () => {
           }),
         ].join('\n')
       )
+      expect(headings).toMatchInlineSnapshot(`
+      Array [
+        Object {
+          "aliases": undefined,
+          "level": 1,
+          "permalinkSlug": "hello-world",
+          "slug": "hello-world",
+          "title": "hello world <a href=\\"wow\\"></a>",
```

Comment:
Do we want the anchor tag to be part of the title? 💭

Reply:
I've thought so hard about this and not 100% sure what the path forward is. I think it's important to not exclude it from the data, but maybe we need two attributes instead of just this one?

- One attribute that contains the content of the heading minus the aliases (what's currently being set as `title`)
- One attribute that contains the plain text of the heading (aka no HTML or aliases)

Might not be our use case now, but if someone wants to use this data to render the headings and whatever HTML is nested, it seems reasonable to have that available. I think our main use case is to have the plain text that we can put in the content of a table of contents. We probably wouldn't want to render an anchor that goes somewhere other than a heading within a table of contents, so now I think this needs expanded upon.

Reply:
My initial take here is that title should be the text content of heading and maybe we don't need an additional attribute with the full markup? I'm having trouble thinking of a scenario where we would want the full markup with the anchor tags, but I could definitely be missing something you're thinking of!

We could always add this additional attribute with the full markup down the road if we come up with a need for it.

Reply:
You're right! Adding another attribute should be super easy if the use case arises. 👏🏻  I'm going to update this to just be the plain text content of the heading.

Reply:
Updated!